keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -819,6 +820,42 @@ namedResources:
   - gpu-1
 ```
 
+### Optional allocation
+
+The default used by the examples above is to allocated exactly one named
+resource instance for each request. A range may be specified to request more
+than one or, if this would be acceptable, none at all. Each allocated instance
+will be listed in the allocation result, same as when no range is
+specified. That list may be empty if nothing was allocated.
+
+This example asks for all GPUs with more than 32Gi that are available:
+```
+driverRequests:
+- driverName: cards.dra.example.com
+  requests:
+  # Each entry here is a request for one resource.
+  - namedresources:
+      range:
+        minimum: 1
+      selector: |-
+        attributes.quantity["memory"].isGreaterThan(quantity("32Gi"))
+```
+
+This example allows to run a pod with a GPU when one is available and without
+one if not:
+```
+driverRequests:
+- driverName: cards.dra.example.com
+  requests:
+  # Each entry here is a request for one resource.
+  - namedresources:
+      range:
+        minimum: 0
```

Comment:
is it possible to have minimum == maximum?

Reply:
May be add it to the KEP too

Reply:
As an example or in the `NamedResourcesRange` definition below?

Reply:
Added.